packaging/os/pacman.py

```diff
@@ -175,12 +170,6 @@ def upgrade(module, pacman_path):
         module.exit_json(changed=False, msg='Nothing to upgrade')
 
 def remove_packages(module, pacman_path, packages):
-    if module.params["recurse"]:
```

Comment:
Good call, but `recuse=yes` support needs to be fixed. It was in my todo, but feel free to take it up :)
FWIW, `force=yes` isn't really same as `recurse=yes`.